lms/djangoapps/grades/tests/test_transformer.py

```diff
@@ -330,6 +353,38 @@ def test_course_version_collected_in_split(self):
         block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
         self.assertIsNotNone(block_structure.get_xblock_field(blocks[u'course'].location, u'course_version'))
 
+    def test_grading_policy_collected(self):
+        # the calculated hash of the original and updated grading policies of the test course
+        expected_original_grading_policy_hash = u'ChVp0lHGQGCevD0t4njna/C44zQ='
+        expected_updated_grading_policy_hash = u'TsbX04qWOy1WRnC0NHy+94upPd4='
```

Comment:
Tiny nit: I'm in favor of leaving the `expected_` prefix off of these variable names